pkg/rule/rule_data.go

```diff
@@ -21,3 +23,16 @@ func NewRuleData(id []byte, exprs []expr.Any) RuleData {
 		ID:          id,
 	}
 }
+
+func (d RuleData) getCounters() (bytes *int64, packets *int64, error error) {
+	for _, ex := range d.Expressions {
+		switch v := ex.(type) {
+		case *expr.Counter:
+			bytes := int64(v.Bytes)
```

Comment:
Looks like these should be `uint64` already so I don't think we need to convert https://github.com/google/nftables/blob/main/expr/counter.go#L26

Reply:
Hmmm it's in `metric.Count()` that int64 is expected, so I think this conversion has to happen somewhere from the uint64 that comes out of the expression
https://github.com/ngrok/firewall_toolkit/blob/ce22f891bcd8f58b0d9e1f288fff9e3d438b1604/pkg/metrics/metrics.go#L10-L11

Reply:
Oh whoops, I forgot about that. Thanks!